controllers/ChangePassword.js

```diff
@@ -0,0 +1,50 @@
+import bcrypt from 'bcrypt';
+import { User } from '../models';
+
+/**
+ * Change user password
+ */
+class ChangePassword {
+  /**
+   * @description Change user password
+   * @param {object} req http request object
+   * @param {object} res http response object
+   * @returns {object} response
+   */
+  static async changePassword(req, res) {
+    const passwordResetToken = req.id;
+    const { password } = req.body;
+
+    try {
+      // save new password to db
+      await User.update(
+        { password: bcrypt.hashSync(password, bcrypt.genSaltSync(10)) },
+        { where: { passwordResetToken } }
+      );
+    } catch (error) {
+      return res.status(500).json({
+        success: false,
+        errors: [error.message]
+      });
+    }
+
+    try {
+      // delete password token
+      await User.update(
```

Comment:
Hi. Please consider using `User.destroy()` to delete the passwordReserToken.
https://lorenstewart.me/2016/10/03/sequelize-crud-101/ 
http://docs.sequelizejs.com/class/lib/model.js~Model.html#instance-method-destroy

Reply:
`User.destroy` will delete the entire row. I'm not trying to delete the row on that line, only trying to remove the password token